### PR TITLE
Toxicity classifier refinery

### PR DESCRIPTION
**Pull request by LeonardPuettmannKern**
PR checklist:
- [x] Tested by creator on localhost:8000/docs
- [x] Tested by creator on refinery
- [x] Tested by reviewer on localhost:8000/docs
- [x] Tested by reviewer on refinery
- [x] (If added) common code tested in notebook/ script
- [x] Conforms with the agreed upon code pattern

### CODE REVIEW

File: classifiers/llm/bert_toxicity_classifier/code_snippet_refinery.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import json
+
+API_KEY: str = "<API_KEY_GOES_HERE>"
+ATTRIBUTE: str = "headline" # only text attributes
+
+def bert_toxicity_classifier(record):
+    """
+    Uses toxic-bert via Hugging Face Inference API to classify toxicity in text.
+    """
+    api_token = API_KEY
```
Comment (DerKernigeFeuerpfeil) [inline classifiers/llm/bert_toxicity_classifier/code_snippet_refinery.md:12]: could not test as my HF key dsnt work for some reason

also, this makes it a premium I think?

File: classifiers/llm/bert_toxicity_classifier/config.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+def get_config():
+    return build_classifier_premium_config(
+        function=bert_toxicity_classifier,
+        input_example=INPUT_EXAMPLE,
+        issue_id=80,
+        tabler_icon="MoodAngry",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        gdpr_compliant="true",
+        type="python_function",
```
Comment (DerKernigeFeuerpfeil) [inline classifiers/llm/bert_toxicity_classifier/config.py:15]: I think this must be premium

File: classifiers/llm/bert_toxicity_classifier/config.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+def get_config():
+    return build_classifier_premium_config(
+        function=bert_toxicity_classifier,
+        input_example=INPUT_EXAMPLE,
+        issue_id=80,
+        tabler_icon="MoodAngry",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        gdpr_compliant="true",
```
Comment (DerKernigeFeuerpfeil) [inline classifiers/llm/bert_toxicity_classifier/config.py:14]: this true? Is huggingface generally GDPR compliant?

**Review by DerKernigeFeuerpfeil — CHANGES_REQUESTED** [review]
minor changes, also I could not validate the functionality due to my API key not working. If you want another review @LeonardPuettmannKern , please DM me

File: classifiers/llm/bert_toxicity_classifier/config.py (hunk 2 above)
Comment (LeonardPuettmannKern) [reply classifiers/llm/bert_toxicity_classifier/config.py:15]: Oh yes, you're right!

File: classifiers/llm/bert_toxicity_classifier/config.py (hunk 3 above)
Comment (LeonardPuettmannKern) [reply classifiers/llm/bert_toxicity_classifier/config.py:14]: Probably not! Will change.

File: classifiers/llm/bert_toxicity_classifier/code_snippet_refinery.md (hunk 1 above)
Comment (LeonardPuettmannKern) [reply classifiers/llm/bert_toxicity_classifier/code_snippet_refinery.md:12]: True!

**Review by DerKernigeFeuerpfeil — APPROVED** [review]

**Pull request merged**